src/carim_discord_bot/discord_client/arguments.py

```diff
@@ -268,24 +268,24 @@ async def process_user_message_args(channel_id, parsed_args):
             for r in result.get('users', list()):
                 if stats is None:
                     stats = tuple(k for k in r.keys() if k not in ('cftools_id', 'rank', 'latest_name'))
-                    result_data.append([stat for stat in ('#',) + stats + ('name',)])
+                    result_data.append([stat for stat in ('#',) + ('name',) + stats])
                 line_items = [r['rank']]
                 for stat in stats:
+                line_items += [r['latest_name']]
```

Comment:
This would need to go before the for loop

Reply:
Thanks. Moved.